pycbc/inference/likelihood.py

```diff
@@ -718,6 +723,21 @@ def loglr(self, **params):
                 # if the waveform terminates before the filtering low frequency
                 # cutoff, there is nothing to filter, so just go onto the next
                 continue
+            if self.calib:
+                cal_init = {}
+                init_keys = ["a_tst0", "a_pu0", "c0", "d0", "freq", "fc0",
+                             "fs0", "qinv0"]
+                init_vals = [self.tfs[det][i] for i in range(6)] + [6.9, 0.05]
+                cal_init.update({i: v for i, v in zip(init_keys, init_vals)})
```

Comment:
Why are you picking these values? The parameters should be MCMC'd over so you need to be getting their values from ``params`` somehow here.